sdk/keyvault/keyvault-secrets/package.json

```diff
@@ -87,7 +88,6 @@
     "@types/fs-extra": "^8.0.0",
     "@types/mocha": "^5.2.5",
     "@types/node": "^8.0.0",
-    "@types/query-string": "6.2.0",
```

Comment:
same in keyvault-certificates

Reply:
I'm not updating certificates as part of this PR